languages/elixir/config.json

```diff
@@ -146,7 +146,7 @@
       {
         "slug": "structs",
         "uuid": "d8d25b93-d918-421e-bb57-cbdaf428d9e2",
-        "concepts": ["structs", "static-access-operator"],
```

Comment:
While trying to write something in its about.md, I came to the conclusion that the static access operator is a much smaller topic than all the other concepts and I would want to propose to treat it as part of the structs concept.